Sources/MapboxDirections/RouteAlert.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  RouteAlert.swift
+//  MapboxDirections
+//
+//  Created by Avi Cieplinski on 10/6/20.
+//  Copyright © 2020 Mapbox. All rights reserved.
+//
+
+import Foundation
```

Comment:
I think we do not use Copyright header sections across this repo, so probably better to remove this one as well.

Reply:
This file isn't needed and I'll remove it.